From 0d245c75da69a440c1981a68a981cb03fe3283de Mon Sep 17 00:00:00 2001
From: Will Drewry <wad@chromium.org>
Date: Mon, 16 Apr 2012 16:56:17 -0500
Subject: [PATCH 787/988] CHROMIUM: arch/arm: select HAVE_ARCH_SECCOMP_FILTER

(I will post this upstream after the 3.5 merge window)

Reflect architectural support for seccomp filter.

Change-Id: I163078260e73a8fb7b9967ce740bd21f83902b8e
Signed-off-by: Will Drewry <wad@chromium.org>

BUG=chromium-os:27878
TEST=compiles for arm. Need to test on a live machine.

Change-Id: Ic0286cc7d150838fbfa05e259ea908aeeef1b864
Reviewed-on: https://gerrit.chromium.org/gerrit/21377
Reviewed-by: Will Drewry <wad@chromium.org>
Tested-by: Will Drewry <wad@chromium.org>
Signed-off-by: Sasha Levitskiy <sanek@google.com>
---
 arch/arm/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 403e876..2dca5ab 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -30,6 +30,8 @@ config ARM
 	select HAVE_HW_BREAKPOINT if (PERF_EVENTS && (CPU_V6 || CPU_V6K || CPU_V7))
 	select HAVE_C_RECORDMCOUNT
 	select HAVE_GENERIC_HARDIRQS
+	select HAVE_SPARSE_IRQ
+	select HAVE_ARCH_SECCOMP_FILTER
 	select GENERIC_IRQ_SHOW
 	select CPU_PM if (SUSPEND || CPU_IDLE)
 	select GENERIC_PCI_IOMAP
-- 
2.10.0

